src/collective/mustread/interfaces.py

```diff
@@ -152,36 +150,40 @@ def most_read(days=None, limit=None):
         :rtype: Generator
         '''
 
-    #  ------- @frisi --- TO BE IMPLEMENTED ---------------------------------
-
-    def schedule_must_read(obj, userids, deadline=None):
-        '''
-        Schedule that an object must be read by some users before a deadline.
+    def schedule_must_read(obj, userids, deadline, by=None):
+        '''Schedule that an object must be read by some users before a deadline.
 
         Calling this method is optional. An object does not have to be
         scheduled as 'must-read' in order to track reads.
 
         Calling this method and scheduling an object as 'must-read' enables
         tracking of 'unread' status for specific sets of users and querying
-        for those via the ``who_unread`` method.
+        for those via the ``who_must_read`` method.
 
         :param obj: Object that should be read by users
         :type obj: Content object (must be IUUID resolvable)
         :param userids: Userids of the users that should view the object.
-        :type userid: List
+        :type userids: List
         :param deadline: Deadline before which users should view the object.
         :type deadline: datetime
+        :param by: Userid that scheduled the must read request
+        :type by: string
+        :returns: List of userids a mustread deadline has been scheduled for
+                  (existing deadlines do not get changed)
```

Comment:
i guess it's better to not allow re-scheduling of existing read requests.
instead the frontend needs to delete existing mustreads and create new ones.